Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -568,15 +568,15 @@ ThrowCompletionOr<void> CreateVariable::execute_impl(Bytecode::Interpreter& inte
             return vm.throw_completion<InternalError>(TRY_OR_THROW_OOM(vm, String::formatted("Lexical environment already has binding '{}'", name)));
 
         if (m_is_immutable)
-            return vm.lexical_environment()->create_immutable_binding(vm, name, vm.in_strict_mode());
+            return vm.lexical_environment()->create_immutable_binding(vm, name, m_force_strict || vm.in_strict_mode());
```

Comment:
```suggestion
            return vm.lexical_environment()->create_immutable_binding(vm, name, m_force_strict);
```

I haven't seen any instances where the spec passes different values to `createMutableBinding`/`createImmutableBinding` depending on whether the script itself is running in strict mode. Do you think we can remove the RHS?

FWIW, I tried it out and it didn't impact any test result in either direction.

Reply:
Hmm good point. I do remember changing this another way at some stage, and it did break some test262 tests. When you changed this around, did you test this with test-js, or test262? Probably I will need to go through and audit each of the places that this is instruction is invoked.

Reply:
I tested both

Reply:
Right, thanks for that :^) As far as I can tell, you're right, it shouldn't be needed :)